lib/jazzy/sourcekitten.rb

```diff
@@ -168,6 +171,14 @@ def self.doc_coverage
         (@undocumented_tokens.count + @documented_count)
     end
 
+    def self.deduplicate_declarations(declarations)
+      declarations.group_by { |d| [d.usr, d.type.kind] }.values.map do |decls|
```

Comment:
can you move `declarations.group_by { |d| [d.usr, d.type.kind] }.values` into a local variable? would help with legibility IMO.